data/author

```diff
@@ -19,11 +20,13 @@ Clipboard:Shadowfacts
 ComputerCraft:dan200
 Crackpack:FTB
 Crash Landing:Iskandar
+CraftTweaker:Jared
```

Comment:
Not sure if I should feel honored or not >.>

Reply:
You should